apps/wallet-mobile/src/Dashboard/Dashboard.tsx

```diff
@@ -114,16 +125,16 @@ export const Dashboard = () => {
                 disableWithdraw
               />
             )}
-          </Row>
+          </View>
 
           {stakingInfo?.status === 'staked' && (
-            <Row>
+            <View style={styles.row}>
               <StakePoolInfos />
-            </Row>
+            </View>
           )}
         </ScrollView>
 
-        <Actions>
```

Comment:
favor readability

Reply:
I'd like to disagree that creating a component whose only purpose is to add a style is somehow more readable.
Specially considering the new component needs to call the useStyles hook to get that style.

So it's either

```
        <Actions>
          ...
        </Actions>
        
        const Actions = (props: ViewProps) => {
          const { styles } = useStyles()
          return <View {...props} style={styles.actions} />
        }
```

Or 
```
        <View style={styles.actions}>
          ...
        </View>
```

I believe the second one is more readable since:

- There's no `Actions` component that you have to figure out what it does (which is actually nothing)
- You don't have to create a nothing component that adds more lines to the file
- It's clear that this is a View with an `actions` style

Reply:
- almost everything is View, how fast can you read <Head><Body><Content>, and how fast can you read <View styles={[styles.background, styles.head, styles.padding]}> <View styles={[styles.padding, styles.container, styler...MAYBE IS THE BODY?]}>
- with tiny specialization, in just a single read you can figure out what is on screen, View is implementation, and implementation "doesn't" (quotes ofc) matter. 
- this tiny specialization are so important that are even libraries designed for that, such as styled-components. (which I think we should consider later)
- imho, at least for me, I can read and understand 10x faster tiny specialization, over a lot of views. - that's why prioritize readability

Reply:
Performance in these cases are just irrelevant. (there are libraries that do 10x more overhead to accomplish the same) I'm more concern about our sync issues over the renderings.
The first paragraph I didn't get it. 
About the philosophy, we are not restricting anything, but if a good soul already abstracted the Views aways, favouring readability, there is absolutely no reason to throw that away.

Reply:
The reason I changed  is that this file was not converted for themes and was using `styles` directly. So this was the component:

```
const Actions = (props: ViewProps) => <View {...props} style={styles.actions} />
```

Since `styles` can't be accessed now without useStyles, that's why I removed the abstraction.

Reply:
yap, I got it, but I think we shouldn't drop it, my suggestion is too consume the styles here and keep as it was before `<Row..` and `<Actions`, we don't need to code everything this way, but since it's there, it would be nice to don't drop it.
later with styled-components it will be simpler or at least more "beautiful" to accomplish it.